features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1780,6 +1782,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }, [ initialValues ]
     );
 
+    useEffect(
+        () => {
+            let bindingType: string = SupportedAccessTokenBindingTypes.NONE;
+
+            if (initialValues?.accessToken?.bindingType) {
+                bindingType = initialValues.accessToken.bindingType;
+            } else if (isFAPIApplication) {
+                bindingType = SupportedAccessTokenBindingTypes.CERTIFICATE;
+            } else if (metadata?.accessTokenBindingType?.defaultValue) {
+                bindingType = metadata.accessTokenBindingType.defaultValue;
+            }
+            setCurrentBindingType(bindingType);
+        }, [ initialValues?.accessToken?.bindingType,
+            metadata?.accessTokenBindingType?.defaultValue,
+            isFAPIApplication ]
+    );
```

Comment:
Formatting improvement.
```suggestion
    useEffect(() => {
        let bindingType: string = SupportedAccessTokenBindingTypes.NONE;

        if (initialValues?.accessToken?.bindingType) {
            bindingType = initialValues.accessToken.bindingType;
        } else if (isFAPIApplication) {
            bindingType = SupportedAccessTokenBindingTypes.CERTIFICATE;
        } else if (metadata?.accessTokenBindingType?.defaultValue) {
            bindingType = metadata.accessTokenBindingType.defaultValue;
        }
        setCurrentBindingType(bindingType);
    }, [
        initialValues?.accessToken?.bindingType,
        metadata?.accessTokenBindingType?.defaultValue,
        isFAPIApplication
    ]);
```